docs/automations/automations.md

```diff
@@ -0,0 +1,11 @@
+# Automations
+
+Hubspot gives you the ability to automate workflows to streamline your pipeline process. You can use this to rotate agencies, create deals, automate tasks, and manage your data. Automations can also be used to trigger internal notifications so that team members can execute next actions or follow ups.
+
+## Current Automations
+
+Used in Contracts Outreach Pipeline
+
+- Auto assign Laney to a task to update Contracts in Hubspot / Contract in Airtable
```

Comment:
If we want to add in more examples we now also have:
Automatically notify Laney when an agency enters "Routed to Scheduling" in Support Pipeline
Auto assign Laney to a task to delete contact when Spam is received

Reply:
*added suggested change below if we want to add those in